Description: Add -logg and -lm to LDADD flags to fix FTBFS with binutils-gold.
Bug-Debian:
 http://bugs.debian.org/556483,
 http://bugs.debian.org/713479
Author: Alessio Treglia <alessio@debian.org>
Forwarded: no
Last-Update: 2013-07-06

--- a/src/Makefile.am
+++ b/src/Makefile.am
@@ -39,7 +39,7 @@
 	${vorbis_cflags} -DDATADIR='"$(datadir)/${PACKAGE_NAME}"'
 
 tagtool_LDFLAGS = -export-dynamic
-tagtool_LDADD = $(GTK_LIBS)
+tagtool_LDADD = $(GTK_LIBS) -logg -lm
 
 tagtool_SOURCES = file_list.c file_list.h file_util.c file_util.h	\
 	edit_tab.c edit_tab.h elist.c elist.h main.c math_util.c	\
